README.md

```diff
@@ -31,7 +31,7 @@ _Follow steps as necessary. If you already have Reaction installed, you may be a
 
     You'll need to create a docker network for the GraphQL service and the Reaction Storefront to communicate
     ```
-    docker network create reaction-api
+    docker network create api
```

Comment:
This would need to be `docker network create api.reaction.localhost`

Reply:
Hmm. Based on my testing yesterday, it seemed docker wanted to consistently use just a flat name and only for the external bit did we use the FQDN. I'm pretty sure what we use here needs to match the `network:` field in the `docker-compose.yml` files.

Reply:
@ticean ? You had previously mentioned in Slack that the command would be `docker network create api.reaction.localhost`